La música cambia con el tilemap elegido + refactor de Resources

eljorgemo opened the pull request:
No he conseguido quitar los errores que me daba con la clase string (tampoco ha hecho falta, lo he conseguido hacer de otra forma). Si alguien se quiere poner a dejarlo todo más bonito en forma de array de strings, para que se pueda cambiar todo utilizando sólo un entero, que me avise, que tengo un par de ideas en mente.
También he hecho bastantes cambios en nombres de resources y borrado archivos provisionales. He ordenado el enum de TexturesId por tipo de textura y alfabéticamente, para que sea más fácil encontrar algo y organizarse.
**TL;DR**: ahora basta con cambiar el string del tilemap para que también cambie la música.

eljorgemo committed:
La música cambia con el tilemap elegido

EstebenR commented on src/SourceFiles/PlayState.h:
```diff
@@ -26,12 +26,13 @@ class PlayState : public GameState
 		{"LivingRoom",Resources::LivingRoom},
 		{"BoilerRoom",Resources::BoilerRoom},
-		{"Gimnasio",Resources::SalaGimnasio}
+		{"Gimnasio",Resources::GymRoom}
```
Deberia ser GymRoom en vez de Gimnasio

EstebenR reviewed (approved):
Todo lo demas bem

eljorgemo committed:
Refactorización de nombres

También he añadido un placeholder para el tilemap del gimnasio. Este commit da problemas con la resolución del gimnasio.

eljorgemo committed:
Más refactorización de nombres

eljorgemo committed:
Merge branch 'master' into 172419830-mapas-con-sonido

eljorgemo committed:
Merge branch 'master' into 172419830-mapas-con-sonido

eljorgemo committed:
Más refactorización
